pkg/LinuxGeneralModule/lgm.go

```diff
@@ -465,8 +467,12 @@ func setUpVrf(vrf *infradb.Vrf) (string, bool) {
 	vrf.Metadata.RoutingTable = make([]*uint32, 1)
 	vrf.Metadata.RoutingTable[0] = new(uint32)
 	var routingTable uint32
+	Name := vrf.Name
+	addKey = 0
 	for {
-		routingTable = GenerateRouteTable()
+		// var key interface{}
+		routingTable, _ = RouteTableGen.GetID(Name, 0)
```

Comment:
why the second returned valued is ignored?
why the second arg is always 0?

Reply:
second returned value can be ignored if the second argument is 0. 
second argument is for the reference to the same key.   actually in the routing table id scenario, there is not need for reference hence ignored.
In vendor code, we used reference for ECMP id case

Reply:
then maybe we could have 2 functions `GetID` and `GetIDWithRef`?